docs/README.iOS.md

```diff
@@ -143,51 +143,66 @@ make -j$(getconf _NPROCESSORS_ONLN) -C tools/depends/target/binary-addons ADDONS
 Before you can use Xcode to build Kodi, the Xcode project has to be generated with CMake. CMake is built as part of the dependencies and doesn't have to be installed separately. A toolchain file is also generated and is used to configure CMake.
 
 ### 6.1. Build with Xcode
-Create an out-of-source build directory:
-```
-mkdir $HOME/kodi-build
-```
 
-Change to build directory:
+Change to Kodi's source code directory:
 ```
-cd $HOME/kodi-build
+cd $HOME/kodi
 ```
 
 Generate Xcode project for ARM 64bit (**recommended**):
 ```
-/Users/Shared/xbmc-depends/x86_64-darwin17.5.0-native/bin/cmake -G Xcode -DCMAKE_TOOLCHAIN_FILE=/Users/Shared/xbmc-depends/iphoneos11.3_arm64-target-debug/share/Toolchain.cmake ../kodi
+make -C tools/depends/target/cmakebuildsys GEN=Xcode
 ```
 
-Or generate Xcode project for ARM 32bit:
-```
-/Users/Shared/xbmc-depends/x86_64-darwin15.6.0-native/bin/cmake -G Xcode -DCMAKE_TOOLCHAIN_FILE=/Users/Shared/xbmc-depends/iphoneos9.3_armv7-target/share/Toolchain.cmake ../kodi
-```
+Additional cmake arguments can be supplied via the CMAKE_EXTRA_ARGUMENTS command line variable
+
+Example to generate Xcode project for ARM 32bit:
+````
+make -C tools/depends/target/cmakebuildsys GEN=Xcode CMAKE_EXTRA_ARGUMENTS="-DCMAKE_TOOLCHAIN_FILE=/Users/Shared/xbmc-depends/iphoneos9.3_armv7-target/share/Toolchain.cmake"
+````
 
-**WARNING:** The toolchain file location differs depending on your iOS and SDK version. You have to replace `x86_64-darwin15.6.0-native` and `iphoneos11.3_arm64-target-debug` or `iphoneos9.3_armv7-target` in the paths above with the correct ones on your system.
+**WARNING:** The toolchain file location differs depending on your iOS and SDK version. You have to replace `iphoneos11.3_arm64-target-debug` or `iphoneos9.3_armv7-target` in the paths above with the correct ones on your system.
 
 You can check `Users/Shared/xbmc-depends` directory content with:
 ```
 ls -l /Users/Shared/xbmc-depends
 ```
 
-**Start Xcode, open the Kodi project file** (`kodi.xcodeproj`) located in `$HOME/kodi-build` and hit `Build`.
+**Start Xcode, open the Kodi project file** (`kodi.xcodeproj`) located in `$HOME/kodi/build` and hit `Build`.
 
 **WARNING:** If you have selected a specific iOS SDK Version in step 4 then you might need to adapt the active target to use the same iOS SDK version, otherwise build will fail. Be sure to select a device configuration. Building for simulator is not supported.
 
 ### 6.2. Build with xcodebuild
-Alternatively, you can also build via Xcode from the command-line with `xcodebuild`, triggered by CMake:
+Alternatively, you can also build via Xcode from the command-line with `xcodebuild`:
+
+Change to Kodi's source code directory:
+```
+cd $HOME/kodi
+```
+
+Generate Xcode project for ARM:
+```
+make -C tools/depends/target/cmakebuildsys GEN=Xcode
+```
```

Comment:
project was already generated in section 6.1

Reply:
Is it expected that if someone was doing an xcodebuild they would follow the steps from the xcode build to get to it?

Reply:
yes, otherwise you could remove 6.1, which will lead to the same result (list the configure command once)

Reply:
Ok, ive had others say they would have thought each section was completely independent of the others. Happy to go your way though. 👍

Reply:
Then how to clone, configure depends and build depends would be missing too in this step.

Reply:
point sections, majors naturally follow on 1-2-3-4, etc, but 6.1, 6.2, 6.3 arent all done.
But as i said, happy to put it back to how existing docs are.

Reply:
I looked now at the whole step 6 and think 6.2 should actually be a subsection of 6.1.

Reply:
As a named subsection, or just as a mention (eg. either open xcode or use xcodebuild with the following command) in 6.1 and combine them

Reply:
I think something like, you can use this command to build from terminal instead of opening xcode would be enough.